pkg/storage/s3/s3.go

```diff
@@ -290,7 +351,7 @@ func (d *driver) CreateStorage(cr *imageregistryv1.Config) error {
 		for i := 0; i < 5000; i++ {
```

Comment:
Side note - probably worth adding a comment that we retry if our generated name happens to match an existing s3 bucket.

Reply:
updated in next push